### ISSUE TITLE
Sandstone if statements within loops lead to say() compiling incorrectly

### ISSUE THREAD

**Issue by astrealRBLX** [TheMrZZ/sandstone#94]
```ts
for (let i = 0; i < 5; i++) {
  say('some string') // compiles incorrectly
  kill('@e[tag=xjfjsdf]') // compiles correctly
  setblock(abs(0, 100, 0), 'minecraft:air') // compiles correctly
  _.if(someVariable.equalTo(1), () => { // arbitrary if statement
    
  })
}
```
compiles into
```
say "some string"
kill @e[tag=xjfjsdf]
setblock 0 100 0 minecraft:air
run say "some string"
kill @e[tag=xjfjsdf]
setblock 0 100 0 minecraft:air
run say "some string"
kill @e[tag=xjfjsdf]
setblock 0 100 0 minecraft:air
run say "some string"
kill @e[tag=xjfjsdf]
setblock 0 100 0 minecraft:air
run say "some string"
kill @e[tag=xjfjsdf]
setblock 0 100 0 minecraft:air
```
The first `say()` compiles correctly but the following ones are prefixed with `run` which doesn't work. I only tested `kill()` and `setblock()` besides `say()` but they both appeared to compile correctly.

**Comment by ncfumction** [comment]
The reason is that your `if` doesn't produce any command while sandstone expects commands. Probably should throw an error though.

**Comment by TheMrZZ** [comment]
Fixed in latest version v1.13.3.
